### DISCUSSION TITLE
EKS 1.24 support

### DISCUSSION THREAD

**Discussion by jeffreymlewis** [gruntwork-io/knowledge-base#591] [Feedback]
What's the timeline for EKS 1.24 support?

---

<ins datetime="2022-11-16T16:56:06Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109610">Tracked in ticket #109610</a></p>
</ins>

**Comment by autero1** [comment]
We've started working on it and we're tracking progress in https://github.com/gruntwork-io/terraform-aws-eks/issues/477. While we cannot give an exact time estimate, we want to highlight that this has high priority.
